controllers/spacerequest/spacerequest_controller.go

```diff
@@ -15,18 +16,26 @@ import (
 	"github.com/redhat-cop/operator-utils/pkg/util"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+// TokenRequestExpirationSeconds is just a long duration so that the token will never expire.
+// However, this token will provide access only at the namespace scope, so once the namespace will be deleted this token will just be invalid.
+const TokenRequestExpirationSeconds = 3650 * 24 * 60 * 60
```

Comment:
Thank you for improvement to the comment! I've applied that suggestion! 

The idea behind `CreateTokenRequest` was to keep it "generic", if need to create TokenRequests from other controllers in the future it can be reusable. Hopefully that makes sense.